.github/workflows/config/macos-pkg-choices-freethreaded.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ooops, sorry should have caught this before: can you this to `.github/config/macos-pkg-choices-freethreaded.xml` (i.e. not in `workflows`)?